docs/radial-chart.md

```diff
@@ -67,6 +67,10 @@ SVG paths (which is what the arc series is made up of) have numerous manipulable
 Type: `string`
 The className to be added to an individual arc in the series.
 
+#### padAngle (optional)
+Type: `number|function`
+The padding to be applied between arcs.
```

Comment:
I think including some text here like, "see above donut chart for an example of a padded angle" would be fine